fedimint-api/src/module/mod.rs

```diff
@@ -386,6 +386,7 @@ pub struct ApiVersion {
 #[async_trait]
 pub trait ModuleGen: Debug + Sized {
     const KIND: ModuleKind;
+    const DATABASE_VERSION: DatabaseVersion;
```

Comment:
The version of a database is a property of the module instance.

Reply:
Perhaps this is a misnomer. This represents the database version that the current code works with. So I believe its actually a static property of the code which we will need to bump when there's a change to the db schema of a module.

The database version that is persisted to the database is a property of the module instance though (this is already isolated using the `IsolatedDatabaseTransaction` mechanism.

Reply:
Added comment in `ModuleGen` to clarify its purpose

Reply:
Oh, you're right.